products/memgraph/upcoming/how_to_guides/query-memgraph-programmatically.md

```diff
@@ -128,59 +130,146 @@ public class JavaQuickStart {
 
 #### JavaScript Example
 
-The details about Javascript driver can be found on
+> **Disclaimer**: Running queries directly from a web browser is **not a recommended**
```

Comment:
Because we don't recommend this usage, it may be appropriate to put it as the last example.

Reply:
Definitely makes sense!